R/package.R

```diff
@@ -81,9 +82,14 @@ send_headers <- c("Accept" = "application/vnd.github.v3+json",
 #' gh("/licenses") # error code 415
 #' gh("/licenses",
 #'    .send_headers = c("Accept" = "application/vnd.github.drax-preview+json"))
+#'  
+#' ## Access Github Enterprise API
+#' gh("/user/repos", type = "public", .api_url = "https://github.foobar.edu/api/v3")
```

Comment:
I think it is already inside the existing `\dontrun{}`. (See the new line 88 below -- the closing bracket)